src/test/java/bio/terra/service/filedata/azure/AzureSynapsePdaoConnectedTest.java

```diff
@@ -244,6 +292,48 @@ private void testSynapseQuery(IngestRequestModel ingestRequestModel, String inge
     assertThat(
         "List of names should equal the input", firstNames, equalTo(List.of("Bob", "Sally")));
 
+    // 5 - Create external data source for the snapshot
+    // where we'll write the resulting parquet files
+    String parquetSnapshotLocation =
+        String.format("https://%s.blob.core.windows.net", snapshotStorageAccountResource.getName());
+    BlobUrlParts snapshotSignUrlBlob =
+        azureSynapsePdao.getOrSignUrlForTargetFactory(
```

Comment:
This is going to cause conflicts with https://github.com/DataBiosphere/jade-data-repo/pull/1056 because `getOrSignUrlForTargetFactory` is being moved to `AzureBlobStorePdao`